index.js

```diff
@@ -18,18 +18,14 @@ const kebabCase = '([a-z][a-z0-9]*(-[a-z0-9]+)*)'; // kebab-case
 const camelCase = '([a-z0-9]+([A-Z][a-z0-9]+)*)'; // camelCase
 
 module.exports = {
-  extends: ['stylelint-config-standard', 'stylelint-prettier/recommended'],
+  extends: ['stylelint-config-standard-scss', 'stylelint-prettier/recommended'],
```

Comment:
`stylelint` now does not support non-CSS syntax internally any more and requires extra configuration. We must support both CSS and SCSS. To do so `stylelint-config-standard-scss` has been added. This itself includes `stylelint-config-standard` and so that package has been uninstalled as an explicit dependency, `-scss` will do everything it was already doing.